### PR TITLE
Lua controller

### PR DESCRIPTION
**Pull request by kishtarn555**
This do to get all LuaCode in one single Sript object to allow easy communication between Lua Functions like FurnitureActions and other possibly eg EntityIA

### CODE REVIEW

**Comment by vogonistic** [comment]
I found 4 places where you leave old code commented out. You should probably delete those and let git track that.

**Comment by kishtarn555** [comment]
Old code eliminated

File: Assets/Scripts/Controllers/LuaController.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        object func = getGlobal(functionName);
+        return LuaScripts.Call(func, args);
+    }
+    public DynValue Call(object functionName,params object[] args) {
+        return LuaScripts.Call(functionName, args);
+    }
+    public void SetGlobal(string name, object value) {
+        LuaScripts.Globals[name] = value;
+    }
+    public object getGlobal(string name) {
```
Comment (OzieGamma) [inline Assets/Scripts/Controllers/LuaController.cs]: SetGlobal start with a big letter. So I think this should be GetGlobal

Comment (kishtarn555) [inline Assets/Scripts/Controllers/LuaController.cs]: Fixit Thanks

**Comment by vogonistic** [comment]
I'm not sure what the benefits are. Can you explain?

**Comment by vogonistic** [comment]
Closing this for now. Please reopen if you fix the conflicts.

**Pull request closed without merging**